CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.1.0]
+### Changed
+- Bumped `@metamask/safe-event-emitter` from `2.0.0` to `3.0.0` ([#148](https://github.com/MetaMask/json-rpc-engine/pull/148))
```

Comment:
Looks like we also bumped `@metamask/utils` from `^5.0.1` to `^5.0.2`, do we want to include that as well?

Reply:
Oops sorry I forgot to link the PR. It's this one: https://github.com/MetaMask/json-rpc-engine/pull/151. It looks like this was changed in `package.json`.